<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifierWindow</class>
 <widget class="QDialog" name="ModifierWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="dialog">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>290</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="editMail">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Prenom</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editFirstName">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>250</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Compagnie</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editCompany">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editLocation">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="editUrl">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Location</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Url</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editName">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Modification de donn√©es</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
